### DISCUSSION TITLE
Meal planning: rules based on OR?

### DISCUSSION THREAD

**Discussion by Cantello** [mealie-recipes/mealie#3205] [Q&A] [answered]
I just started with a recipe collection/meal planning solution for our family and while I like mealie very much I cannot figure out how to use meal planning correctly. Two questions:
- When trying to set the meal for the day (really only one meal necessary), I click on `random dinner` to have one dish selected from my recipes. This works well until I use tags and would like mealie to only choose from selected tags (e.g. "pasta" OR "salad"). There is always an errror saying that no suitable recipe could be found. I guess this has to do with the rules, which, apparently, are always AND instead of OR what I would need here. What to do to choose between several tags to select meals? Using different rules with a single tag each did not work either.
- Is there any possibility to auto-populate e.g. a complete week with meals? It's not really that much of a hassle but I thought that I maybe overlooked something...

**Comment by michael-genson** [comment, marked as answer]
This is being implemented in https://github.com/mealie-recipes/mealie/pull/4346
